tests/testcafe/tests.js

```diff
@@ -50,6 +50,35 @@ function prepareClient() {
   document.querySelector('base').parentNode.removeChild(document.querySelector('base'));
 }
 
+function hiddenTest(game, variant, md5, tests) {
+  const testUrl = `${server}/library/${game}.vtt#VTT`;
+  test.after(async t => {
+    await removeGame(t);
+    await t.expect(Selector('#statesOverlay').visible).ok();
+  })(`Public library: ${game} (variant ${variant})`, async t => {
+    await ClientFunction(prepareClient)();
+    await t
+      .pressKey('esc')
+      .click('#statesButton')
+      .setNativeDialogHandler(() => testUrl, { dependencies: { testUrl }})
+      .click('button.prettyButton.link:nth-child(2)')
+      .pressKey('enter')
+      .hover('.roomState')
+      .click(Selector('button.play').nth(variant));
+    await setName(t);
+    await tests(t);
+    await compareState(t, md5);
+  });
+}
+
+function hiddenLibraryButtons(game, variant, md5, buttons) {
+  hiddenTest(game, variant, md5, async t => {
+    for(const b of buttons) {
+      await t.click(`[id="${b}"]`).wait(4000);
```

Comment:
`wait(4000)` should not be necessary.

Reply:
It might not be, but I put it in there to give the timer time to tick away.  I'll double check though.

Reply:
I see. Look at my comment about the timer though. It's a good idea to check if the timer was running by checking `milliseconds > X` in your last button (not sure if it gets enough time to tick) but you'll have to set the `milliseconds` to something predictable after that test (and put the test result into a property so the state comparison can find it).

Reply:
It stops at a specific time.  In the most recent iteration it only goes for a couple of seconds.